uswds/uswds-site
# USWDS-Site - Migration guide: Add note about autoprefixer

# Summary

Moved `Autoprefixer` note to an `include` and added the note to the Migrating to 3.0 page. 

> ~**Warning**~
> ~Ignored two snyk vulnerabilities causing test failures. This can be resolved by #2322~

_Warning resolved and can be ignored. Leaving for historical context._

## Related issue

Closes #2296

## Preview link

[Migrating to 3.0 page →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-migration-autoprefixer/documentation/migration/#3-update-your-sass-compiler-settings-and-recompile-css-2)
[Phase 2: Compile →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-migration-autoprefixer/documentation/getting-started/developers/phase-two-compile//)

[Changelog →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-migration-autoprefixer/documentation/migration/#latest-updates)

> **Note**
> Must update changelog date before merge

## Problem statement

Issue arose from a  [discussion](https://gsa-tts.slack.com/archives/C3F14AHSQ/p1696437418207649?thread_ts=1696434410.341549&cid=C3F14AHSQ) in the public slack channel. Some users may be using the migration guide exclusively to set up their builds. Without this important piece of information, these users may run into issues with their CSS.

## Solution

Use note about Autoprefixer from Phase 2: Compile page in the Sass compiler instructions.

## Testing and review

1. Inspect new `autoprefixer-note.html` does have gramattical errors.
2. Visit the [Phase 2: Compile](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-migration-autoprefixer/documentation/getting-started-for-developers/) page
3. Confirm Autoprefixer note matches what's on [develop](https://designsystem.digital.gov/documentation/getting-started/developers/phase-two-compile/).
5. Visit the [Migrate to 3.0](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-migration-autoprefixer/documentation/migration/) page
6. Inspect new note re: Autoprefixer
7. Confirm placement is appropriate
